packages/autorest.go/src/m4togocodemodel/adapter.ts

```diff
@@ -39,14 +41,14 @@ export async function m4ToGoCodeModel(host: AutorestExtensionHost) {
     
     const codeModel = new go.CodeModel(info, type, session.model.language.go!.packageName, options);
     if (session.model.language.go!.host) {
-      codeModel.host = session.model.language.go!.host;
+      codeModel.host = <string | undefined>session.model.language.go!.host;
     }
     if (session.model.language.go!.module && session.model.language.go!.moduleVersion) {
       codeModel.options.module = new go.Module(session.model.language.go!.module, session.model.language.go!.moduleVersion);
     } else if (session.model.language.go!.module || session.model.language.go!.moduleVersion) {
       throw new Error('--module and --module-version must both or neither be set');
     } else if (session.model.language.go!.containingModule !== '') {
-      codeModel.options.containingModule = session.model.language.go!.containingModule;
+      codeModel.options.containingModule = <string | undefined>session.model.language.go!.containingModule;
```

Comment:
Does it make sense to assign undefined to the containingModule?

Reply:
This is like the above case. I removed the `| undefined`.